wp-config.php

```diff
@@ -8,48 +8,46 @@
  * https://pantheon.io/docs
  */
 
+/**
+ * Pantheon platform settings. Everything you need should already be set.
+ */
+if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+	require_once(dirname(__FILE__) . '/wp-config-pantheon.php');
```

Comment:
nit: I think our coding guideline is two spaces.

Reply:
[WordPress Coding Standard](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation)
> Your indentation should always reflect logical structure. Use real tabs and not spaces, as this allows the most flexibility across clients.

My intention was to match WordPress standards and match `wp-config-sample.php` and keep in line with the rest of WordPress. Current wp-config has all-of-the-above.

Reply:
OK, if WP uses tabs, then it's best to stick to that. Make sure that it's tabs everywhere in the file though.